### ISSUE TITLE
500 + Jeweller's Orb error when no stash tab is selected (1.0.0)

### ISSUE THREAD

**Issue by Vefy81** [Qt-dev/exile-diary#132]
Error:
Not working. I deleted from appdata completely, and re install, but I get an error after 500 seconds with no stash tab is selected
```
TypeError: Cannot read properties of undefined (reading 'Jeweller's Orb')
    at getVendorRecipeValue (C:\Users\Vefy\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:380:56)
    at getBaseTypeValue (C:\Users\Vefy\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:334:24)
    at Object.<anonymous> (C:\Users\Vefy\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:709:51)
    at step (C:\Users\Vefy\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:33:23)
    at Object.next (C:\Users\Vefy\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:14:53)
    at fulfilled (C:\Users\Vefy\AppData\Local\Programs\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:5:58)
```
OS: Windows NT 10.0.19045

**Comment by Qt-dev** [comment]
Interesting. It looks like it's trying to price a 6 link before it has the pricing from poe.ninja.
It might be that it's pricing the 6 link weapon or armor you have on you, but it's weird that it does not have the data from poe.ninja yet. Maybe it's an API change on their side, or it just triggers the stuff too late or something? I thought it did it at the very beginning 

This is the line, for reference for when I look at it next: https://github.com/Qt-dev/exile-diary/blob/main/src/main/modules/ItemPricer.ts#L493C11-L493C65

**Comment by Qt-dev** [comment]
Also, sorry for the delay, I forgot to press "Comment" last week.

**Comment by Qt-dev** [comment]
This should be fixed with v1.4.0, let me know if you ever hit that issue again :)
